sdcm/cluster.py

```diff
@@ -1244,7 +1244,7 @@ def wait_db_up(self, verbose=True, timeout=3600):
     def is_manager_agent_up(self, port=None):
         port = port if port else self.MANAGER_AGENT_PORT
         # When the agent is IP, it should answer an https request of https://NODE_IP:10001/ping with status code 204
-        response = requests.get(f"https://{normalize_ipv6_url(self.ip_address)}:{port}/ping", verify=False)
+        response = requests.get(f"https://{normalize_ipv6_url(self.external_address)}:{port}/ping", verify=False)
```

Comment:
Is `external_address` proprety?

Reply:
i think, ip_address should resolve that

Reply:
@dkropachev 
will it work in case when ip_ssh_connections=ipv6?

Reply:
@Orenef11 , yes, it is

Reply:
> i think, ip_address should resolve that

@aleksbykov , from what I see ip_address relates to inner-cluster communication, while external_address relates to the address that is used to address the node from the outside of the cluster, wich is perfectly fitx for this particular case.

Reply:
> @dkropachev
> will it work in case when ip_ssh_connections=ipv6?

@juliayakovlev , Yes it will:
    @property
    def external_address(self):
        if IP_SSH_CONNECTIONS == "ipv6":
            return self.ipv6_ip_address
        elif IP_SSH_CONNECTIONS == 'public' or Setup.INTRA_NODE_COMM_PUBLIC:
            return self.public_ip_address
        else:
            return self.private_ip_address

Reply:
so, what i found is that the bug is in the `ip_address` property, as it doesn't include `IP_SSH_CONNECTIONS == 'public'`

but other than that, it seems to be exactly the same as `external_address`